nautobot/core/templates/plugin_template/navigation.py-tpl

```diff
@@ -0,0 +1,21 @@
+"""Navigation Items to add to Nautobot for {{ app_name }}."""
+
+from nautobot.extras.plugins import PluginMenuButton, PluginMenuItem
+from nautobot.utilities.choices import ButtonColorChoices
+
+menu_items = (
+    PluginMenuItem(
+        # link='plugins:{{ app_name }}:model',  # A reverse compatible link to follow.
+        # link_text = 'Sample Text',  # Text to display to user.
+        # permissions = [],  # Optional: List of permissions required to display this link.
+        # buttons = (  # Optional: Iterable of PluginMenuButton instances to display.
+            # PluginMenuButton(
+                'plugins:{{ app_name }}:model',  # A reverse compatible link to follow.
+                'Sample Text',  # Text to display to user.
+                'mdi mdi-help-circle',  # Button icon CSS Classes (Currently supports Material Design Icons.)
+                ButtonColorChoices.BLUE,  # Optional: Color for the button.,
+                permissions = []  # Optional: List of permissions required to display this button.
+            )
+        )
+    )
+)
```

Comment:
I'd expect this whole block to be commented out, but you have lines 13-19 not commented, as well as the outer `menu_items` and `PluginMenuItem` elements - is this valid as rendered?

Reply:
I will move to just block commenting in Python. The single line comments got me turned around. No, python didn't like this rendered.